test/features/pagecontent/revisions.js

```diff
@@ -13,6 +13,8 @@ describe('revision requests', function() {
 	var revOk = 642497713;
 	var revDeleted = 645504917;
 	var revRedirect = 591082967;
+    var pageName = 'User:GWicke%2fDate';
```

Comment:
I've jsut noticed the three lines above have tabs instead of spaces. Mind s/^I/    / ?